docs/hopsworksai/index.md

```diff
@@ -0,0 +1,35 @@
+# Hopsworks.ai
+
+Hopsworks.ai is our managed platform for running Hopsworks and the Feature Store
+in the cloud. It integrates seamlessly with third-party platforms such as Databricks,
+SageMaker and KubeFlow.
+
+## Getting started
+
+To get started with deploying a cluster in you Azure or AWS environment, follow these guides: [Azure](azure/getting_started.md), [AWS](aws/getting_started.md).
+
+If you need more details about hopsworks.ai cluster creation steps you can check the following guides: [Azure](azure/cluster_creation.md), [AWS](aws/cluster_creation.md).
+
+## Limiting permissions
+
+Hopsworks.ai requires a set of permissions to be able to manage resources in the user’s cloud account. By default, these permissions are set to easily allow a wide range of different configurations and allow us to automate as many steps as possible. While we ensure to never access resources we shouldn’t, we do understand that this might not be enough for your organization or security policy. The following guides explain how you can reduce the permissions you give to hopsworks.ai by doing some of the cluster creation steps manually:
+
+* [Azure](azure/restrictive_permissions.md)
```

Comment:
I guess we should be consistent on listing of Azure and AWS 
https://github.com/logicalclocks/feature-store-api/pull/257/files#diff-bef27b5191049cfac9fd10b40901df0b482a80decc5481ae15696b9ce93f5c33R9
https://github.com/logicalclocks/feature-store-api/pull/257/files#diff-bef27b5191049cfac9fd10b40901df0b482a80decc5481ae15696b9ce93f5c33R11